.github/workflows/firebase_actions.yml

```
@@ -25,10 +25,27 @@ jobs:
       - name: Install Firebase CLI
         run: curl -sL https://firebase.tools | bash
 
-      - name: NPM install
+      - name: Install NPM dependencies to the firebase folder
```

What do you think about installing all needed npm dependencies in one step? 
```suggestion
      - name: Install NPM dependencies
```